app/src/main/java/org/schabi/newpipe/player/helper/PlayerHelper.java

```diff
@@ -131,15 +131,18 @@ public static String getTimeString(final int milliSeconds) {
         ).toString();
     }
 
+    @NonNull
```

Comment:
Even if it makes reading code harder, it's a good pratice in my opinion, because it helps to solve future NPEs.

Reply:
How would it help to solve future NPEs? If `@NonNull`s and `@Nullable`s are used wisely to trigger the developer attention, then they do help in preventing NPEs as the developer is advised to be more careful. But if they are everywhere then the developer will just ignore them outright when looking at the code, because they are just too many and they carry really little information. @litetex what is your opinion?